clients/admin-ui/src/features/integrations/configure-monitor/MonitorStatusCell.tsx

```diff
@@ -0,0 +1,106 @@
+import { formatDistance } from "date-fns";
+import {
+  AntTag as Tag,
+  AntTooltip as Tooltip,
+  AntTypography as Typography,
+  Divider,
+  Drawer,
+  DrawerBody,
+  DrawerCloseButton,
+  DrawerContent,
+  DrawerHeader,
+  DrawerOverlay,
+  useDisclosure,
+} from "fidesui";
+import { Fragment } from "react";
+
+import { formatDate } from "~/features/common/utils";
+import { MonitorExecutionStatus } from "~/types/api";
+import { MonitorStatusResponse } from "~/types/api/models/MonitorStatusResponse";
+
+import styles from "./MonitorStatusCell.module.scss";
+
+const MonitorStatusCell = ({ monitor }: { monitor: MonitorStatusResponse }) => {
+  const executionRecord = monitor.execution_records?.[0];
+  const { isOpen, onOpen, onClose } = useDisclosure();
+  const lastCompletedTime = monitor.last_monitored
+    ? new Date(monitor.last_monitored)
+    : null;
+
+  const formattedTime = lastCompletedTime
+    ? formatDate(lastCompletedTime)
+    : "N/A";
+
+  const formattedDistance = lastCompletedTime
+    ? formatDistance(lastCompletedTime, new Date(), {
+        addSuffix: true,
+      })
+    : "N/A";
+
+  if (!executionRecord) {
+    return <Tag>N/A</Tag>;
+  }
+  if (executionRecord.status === MonitorExecutionStatus.IN_PROGRESS) {
+    if (!lastCompletedTime) {
+      return <Tag color="info">Scanning</Tag>;
+    }
+
+    return (
+      <Tooltip title={`Last scan: ${formattedDistance}`}>
+        <Tag color="info">Scanning</Tag>
+      </Tooltip>
+    );
+  }
+  if (executionRecord.status === MonitorExecutionStatus.COMPLETED) {
+    return (
+      <Tooltip title={formattedTime}>
+        <Tag color="success" data-testid="tag-success">
+          {formattedDistance}
+        </Tag>
+      </Tooltip>
+    );
+  }
+  if (executionRecord.status === MonitorExecutionStatus.ERRORED) {
+    return (
+      <>
+        <Tag
+          color="error"
+          className={styles.monitorStatusTag}
+          data-testid="tag-error"
+        >
+          <Typography.Text onClick={onOpen} className={styles.errorTagText}>
+            Incomplete
+          </Typography.Text>
```

Comment:
Can we make this more accessible? Any `onClick` not on an `<a href="">` or `<button>` is not going to be keyboard accessible with the tab key focus and enter key to activate it. Using a basic button with the same styles seems to work well.
```suggestion
          <button
            type="button"
            onClick={onOpen}
            className={styles.errorTagText}
          >
            Incomplete
          </button>
```